flexbox/src/main/java/com/google/android/flexbox/FlexboxLayoutManager.java

```diff
@@ -189,8 +206,9 @@ public FlexboxLayoutManager(@FlexDirection int flexDirection) {
      * @param flexDirection the flex direction attribute
      * @param flexWrap      the flex wrap attribute
      */
-    public FlexboxLayoutManager(@FlexDirection int flexDirection,
+    public FlexboxLayoutManager(Context context, @FlexDirection int flexDirection,
             @FlexWrap int flexWrap) {
+        setDisplayMetrics(context);
```

Comment:
This way, the values are not updated when android:configChanges="..." is set for the activity. What about calling this every time but only when necessary? We need these values only when the size is infinite.